build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1573,6 +2112,66 @@ message SymlinkAbsolutePathStrategy {
   }
 }
 
+// Describes support for two-phase caching, specifically
```

Comment:
>two-phase caching [](start = 25, length = 17)

I think that we need to describe the two-phase caching, particularly the motivations of it and how it works.

Reply:
I added a lot of new description to the various 2-phase cache messages in an attempt to fix this problem. Let's see if it makes sense to the readership in the Remote Exec Working Group and iterate as needed.

---
In reply to: [458988670](https://github.com/erikma/remote-apis/pull/1#discussion_r458988670) [](ancestors = 458988670)